campaigns.json

```diff
@@ -666,14 +670,6 @@
     "lumisize": -1, 
     "maxcopies": 1, 
     "resize": "auto", 
-    "secondaries": {
```

Comment:
You should not have removed this secondary.

Reply:
What?

Reply:
Wasn't this the one you said remove the secondary?